Aquality.Selenium.Core/tests/Aquality.Selenium.Core.Tests/Visualization/FormDumpTests.cs

```diff
@@ -77,15 +78,6 @@ public void Should_BePossibleTo_CompareWithDump_WithCustomName_WhenElementSetDif
             Assert.That(customForm.Dump.Compare("Set differs"), Is.GreaterThan(0), "Difference with current page should be greater than zero if element set differs");
         }
 
-        [Test]
-        public void Should_BePossibleTo_SaveAndCompareWithDump_WithCustomName_WhenAllElementsSelected()
-        {
-            var customForm = new WebForm();
-            customForm.SetElementsForDump(WebForm.ElementsFilter.AllElements);
-            Assert.DoesNotThrow(() => customForm.Dump.Save("All elements"));
-            Assert.That(customForm.Dump.Compare("All elements"), Is.GreaterThan(0), "Some elements should be failed to take image, so count of elements would not match");
```

Comment:
I know that I suggested this deletion, but I just thought that we can restore it and replace just the last check ...

Reply:
updated